solana/pyth2wormhole/client/src/cli.rs

```diff
@@ -83,6 +88,19 @@ pub enum Action {
             default_value = "20"
         )]
         confirmation_timeout_secs: u64,
+        #[clap(
+            short = 'm',
+            long,
+            help = "Address to use for serving Prometheus metrics.",
+            default_value = "[::]:3000"
+        )]
+        metrics_bind_addr: SocketAddr,
+        #[clap(
+            long,
+            help = "Subpage name for serving Prometheus metrics on http://<metrics_bind_addr>/<metrics_subpage>",
+            default_value = "metrics"
+        )]
+        metrics_subpage: String,
```

Comment:
I got a little confused by this because this is also hardcoded in some places. I think it's better to have it hard-coded as `metrics` as that's the usual path and we really don't need to configure it (I don't see any usacase)